Production/python/Production.py

```diff
@@ -69,16 +74,22 @@
 if options.eventList != '':
     process.source.eventsToProcess = cms.untracked.VEventRange(re.split(',', options.eventList))
 
-import RecoTauTag.RecoTau.tools.runTauIdMVA as tauIdConfig
+tauAtMiniTools.addTauReReco(process)
+tauAtMiniTools.adaptTauToMiniAODReReco(process, options.reclusterJets)
+
+process.combinatoricRecoTaus.builders[0].signalConeSize = cms.string('max(min(0.2, 4.528/(pt()^0.8982)), 0.03)') ## change to quantile 0.95
+process.selectedPatTaus.cut = cms.string('pt > 18.')   ## remove DMFinding filter (was pt > 18. && tauID(\'decayModeFindingNewDMs\')> 0.5)
+
+tauIdConfig = importlib.import_module('TauMLTools.Production.runTauIdMVA')
 updatedTauName = "slimmedTausNewID"
 tauIdEmbedder = tauIdConfig.TauIDEmbedder(
     process, cms, debug = False, updatedTauName = updatedTauName,
-    toKeep = [ "2017v2", "dR0p32017v2", "newDM2017v2", "deepTau2017v2p1", "deepTau2017v2p1", "againstEle2018",  ]
+    toKeep = [ "2017v2", "dR0p32017v2", "newDM2017v2", "deepTau2017v2p1"]
```

Comment:
If `2017v2`, `dR0p32017v2` or `newDM2017v2` can be rerun, they should be stored in the root-tuple.

Reply:
done, see https://github.com/cms-tau-pog/TauMLTools/pull/13/commits/e26149c6cd6c154593046ad5775c1e201722ec20